fal/controllers/bench_swaps.py

```diff
@@ -18,86 +19,78 @@
 config = configparser.ConfigParser()
 config.read("config.ini")
 
-active_len = config.getint("season info", "num-active-on-team")
 
-PARAMS = ""
+class BenchSwaps(Controller):
+    MAL_PARAMS = ""
 
+    def get_swaps(
+        self,
+        session: Session,
+        page: int = config.getint("weekly info", "bench-swap-page-start"),
+        actions: List[Tuple] = [],
+    ) -> List[Tuple]:
+        """
+        NOTE: copied from old FAL script and converted to Python 3 and uses our modern stack
 
-def get_swaps(
-    session: Session,
-    page: int = config.getint("weekly info", "bench-swap-page-start"),
-    actions: List[Tuple] = [],
-) -> List[Tuple]:
-    """
-    NOTE: copied from old FAL script and converted to Python 3 and uses our modern stack
+        Collects swaps for the current week.
 
-    Collects swaps for the current week.
+        This function accesses the swap thread.
 
-    This function accesses the swap thread.
+        @param page page identification for threads
+        @param actions list of tuples (username, post content)
 
-    @param page page identification for threads
-    @param actions list of tuples (username, post content)
+        @return list of tuples (username, post content = swap)
+        """
+        header = dict(config.items("myanimelist.net request header"))
 
-    @return list of tuples (username, post content = swap)
-    """
-    header = dict(config.items("myanimelist.net request header"))
-
-    with mfalncfm_main.session_scope() as session:
-        secret_request_headers = (
-            session.query(Secret)
-            .filter(Secret.context == "myanimelist.net request header")
-            .all()
+        with mfalncfm_main.session_scope() as session:
```

Comment:
Do we need this since session is being passed in as a parameter?

Reply:
nope, good catch